code/modules/reagents/Chemistry-Goon-Pyrotechnics.dm

```diff
@@ -25,10 +25,17 @@
 	result_amount = 4
 	required_temp = 424
 
+datum/reagent/clf3/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	M.adjustFireLoss(1*REM)
+	..()
+	return
+
 /datum/chemical_reaction/clf3/on_reaction(var/datum/reagents/holder, var/created_volume)
 	var/turf/simulated/T = get_turf(holder.my_atom)
-	for(var/turf/simulated/turf in orange(1,T))
+	for(var/turf/simulated/turf in range(1,T))
 		new /obj/effect/hotspot(turf)
+	new /obj/effect/hotspot(T)
```

Comment:
If you're using range(), you don't use this line, since range() already includes your own turf. That's the difference between range and orange.